target/ppc/translate/vmx-impl.c.inc

```diff
@@ -1257,6 +1257,80 @@ static void gen_vsldoi(DisasContext *ctx)
     tcg_temp_free_i32(sh);
 }
 
+static bool trans_VSLDBI(DisasContext *ctx, arg_VN *a)
+{
+    TCGv_i64 t0, t1, t2;
+
+    REQUIRE_INSNS_FLAGS2(ctx, ISA310);
+    REQUIRE_VECTOR(ctx);
+
+    t0 = tcg_temp_new_i64();
+    t1 = tcg_temp_new_i64();
+    t2 = tcg_temp_new_i64();
```

Comment:
Maybe move ``t2 = tcg_temp_new_i64()`` and ``tcg_temp_free_i64(t2)`` inside the ``if (a->sh != 0)`` block?

Reply:
Fixed in https://github.com/PPC64/qemu/pull/18/commits/69a1f824a0db9d22f122f26e2ed86da0b34e52ad